mono/utils/mono-threads-coop.c

```diff
@@ -273,13 +320,13 @@ mono_threads_core_end_global_suspend (void)
 void*
 mono_threads_enter_gc_unsafe_region (void)
 {
-	return mono_threads_reset_blocking_start ();
+	return mono_threads_reset_blocking_start (0);
 }
 
 void
 mono_threads_exit_gc_unsafe_region (void *regions_cookie)
 {
-	mono_threads_reset_blocking_end (regions_cookie);
+	mono_threads_reset_blocking_end (regions_cookie, 0);
```

Comment:
Yep don't know how I missed this one...

Anyway passing zero when the argument is a pointer is perfectly fine, 0 is defined as a null pointer constant in C. Do you mean to use NULL instead for code guidelines / clarity reasons?